MergerLogic/ImageProcessing/TileMerger.cs

```diff
@@ -48,50 +41,38 @@ public TileMerger(ITileScaler tileScaler, IImageFormatter imageFormatter)
                         {
                             mergedImage.ColorSpace = ColorSpace.sRGB;
                             mergedImage.ColorType = mergedImage.HasAlpha ? ColorType.TrueColorAlpha : ColorType.TrueColor;
-                            this._imageFormatter.CovertToFormat(mergedImage,format);
+                            this._imageFormatter.CovertToFormat(mergedImage, format);
                             var mergedImageBytes = mergedImage.ToByteArray();
+
+                            for (var i = images.Count - 1; i >= 0; i--)
+                            {
+                                images[i].Dispose();
```

Comment:
As we use the following line _using (var imageCollection = new MagickImageCollection())_ i think the dispose is already called, please check

Reply:
It seems that the internal images are disposed as well:
https://github.com/dlemstra/Magick.NET/blob/adafa865c5651d69c6e432e13141798ae25626d8/src/Magick.NET/MagickImageCollection.cs#L422
I will remove this.